src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+    private String block;
+    
+    public Block(String blockNumber){
+        this.block = blockNumber;
+        }
+        
+        public String getBlock() {
```

Comment:
Indentation should be done according to coding standard provided in the handbook.